doc/src/tutorials/creating-packages.md

```diff
@@ -605,9 +605,20 @@ may fit your package better.
 
      * `CPLEX.jl` wraps the `CPLEX` library, which can be identified easily in a web search.
      * `MATLAB.jl` provides an interface to call the MATLAB engine from within Julia.
+
 7. Avoid naming a package closely to an existing package
      * `Websocket` is too close to `WebSockets` and can be confusing to users. Rather use a new name such as `SimpleWebsockets`.
 
+8. Avoid using a distinctive name that is already in use in another unrelated project.
+     * Don't use the names `Tkinter.jl`, `TkinterGUI.jl`, etc. for a package that is unrelated
+       to the popular `tkinter` python package. Users are likely to assume an affiliation that
+       does not exist.
+     * It's okay to name a package `HTTP.jl` even though it is unrelated to the popular rust
+       crate `http` because "http" is not a distinctive name.
```

Comment:
Maybe 
```suggestion
       crate `http` because "http" is not a distinctive name, being a more general standard.
```
for some extra clarity on why it isn't a distinctive name might be useful?  Not sure if this is the best wording though.

Reply:
or "because both packages' names refer to the well-known HTTP standard rather than to one another"

Reply:
I tried to pick a verbiage that highlight what users think the name refers to rather than what the name actually refers to, happy to keep looking for more improvements.

Reply:
That sounds good to me!